panic_handler/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "panic_handler"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+version = "0.1.0"
+description = """Default panic handler for libtock. Calls low-level debug on panic and yields."""
+edition = "2018"
+license = "Apache-2.0 OR MIT"
+repository = "https://www.github.com/tock/libtock-rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
I prefer to remove the comment pointing to `cargo`'s documentation from `Cargo.toml`.

Reply:
Done.